tests/Issues/Issue1963Test.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace Issues;
+
+use Mpdf\BaseMpdfTest;
+use Mpdf\Mpdf;
+use Mpdf\Output\Destination;
+
+class Issue1963Test extends BaseMpdfTest
+{
+	protected function set_up()
```

Comment:
This method is kind of unnecessary, make a local variable directly in the test method.